python/frontend/cirrus/cirrus/MLJob.py

```diff
@@ -0,0 +1,61 @@
+from abc import ABCMeta, abstractmethod
+
+class MLJob(object):
```

Comment:
Not clear what is the purpose of this MLJob class.
1. It seems to overlap with class BaseTask
2. It is only inherited by the CVJob

I think it makes sense to think how we can make the interface to all the jobs we run (individual experiments or cross validation experiments) roughly uniform. This seems a roughly half baked solution now. We should discuss this.